src/components/mission/Mission.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Spinner } from 'react-spinners-css';
+import style from './Mission.module.css';
+import { displayMissionFunction, missionStatusFunction } from '../../redux/mission/Mission';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+const Mission = () => {
+  // const [isMember, setMember] = useState(false);
+  const dispatch = useDispatch();
+  const missionList = useSelector((state) => state.missionReducer);
+  // const isMember = useSelector((state) => state.missionReducer);
+  // console.log(isMember);
```

Comment:
- [ ] Kindly Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.. 😉 💯